Makefile.devel

```diff
@@ -84,16 +84,17 @@ DEVELOPERS=\
 -u "gklein	Gerwin Klein	gerwin.klein@nicta.com.au" \
 -u "assia	Assia Mahboubi	assia.mahboubi@inria.fr" \
 -u "monnier	Stefan Monnier	monnier@iro.umontreal.ca" \
--u "tews	Hendrik Tews	hendrik@askra.de"
+-u "tews	Hendrik Tews	hendrik@askra.de" \
+-u "steck	Paul Steckler	steck@stecksoft.com"
 
 # PRERELEASE_PREFIX is used to match PRERELEASE_TAG in sed
 # line in tag target below, which edits $(DOWNLOADHTML)
 # The prereltag.txt is kept as a record in the distrib area
 # of the current pre-release version (currently not used explicitly
 # anywhere for web pages/whatever).
-PRERELEASE_PREFIX=4\.4\.1~pre
-PRERELEASE_TAG=4.4.1~pre
-### Formerly: PRERELEASE_TAG=4.4pre$(shell date "+%y%m%d")
+PRERELEASE_PREFIX=5\.0~pre
+PRERELEASE_TAG=5.0~pre
```

Comment:
@psteckler
as discussed in https://github.com/ProofGeneral/PG/pull/127#discussion_r94273448, from the upcoming release it would be better to use a "-git" version suffix for the development version of PG. So could you change this line to `5.0-git` and likewise for other occurrences of PG's version? (in README.md and proof-site.el)